charts/backingservices/README.md

```diff
@@ -109,27 +109,27 @@ srs:
     # set the external Elasticsearch cluster URL and port details below when using an externally managed elasticsearch
     # Ensure that the specified endpoint is accessible from the kubernetes cluster pods.
-    #domain: ""
-    #port: 9200
-    #protocol: https
+    # domain: ""
+    # port: 9200
+    # protocol: https
     # The elasticsearch connection supports three authentication methods: basic authentication , AWS IAM role-based authentication and Elasticsearch secure connection(tls).
-    # set `srs.srsStorage.tls.enabled: true` to enable tls on internal/external elasticsearch cluster.
+    # Set srs.srsStorage.tls.enabled: true to enable the use of TLS-based authentication to your Elasticsearch service whether is it running as an internalized or externalized service in your SRS cluster.
     tls:
-     enabled: false
-    # Specify the certificate name used to connect to elasticsearch when tls.enabled: true and srsStorage.provisionInternalESCluster: false.
+      enabled: false
+    # To specify a certificate used to authenticate an external Elasticsearch service (with tls.enabled: true and srsStorage.provisionInternalESCluster: false), uncomment the following line to specify the TLS certificate name for your Elasticsearch service
     # certificateName: "Certificate_Name"
-    # set `srs.srsStorage.basicAuthentication.enabled: true` to enable basicAuthentication on internal/external elasticsearch.
+    # Set srs.srsStorage.basicAuthentication.enabled: true to enable the use of basic authentication to your Elasticsearch service whether is it running as an internalized or externalized service in your SRS cluster.
     basicAuthentication:
       enabled: true
-    #     when  basic authentication/tls based authentication is configured to your externally-managed Elasticsearch cluster, uncomment and add the
-    #     parameter details: srs.srsStorage.esCredentials.username and srs.srsStorage.esCredentials.password
+    # To configure basic authentication or TLS-based authentication to your externally-managed Elasticsearch service in your SRS cluster, uncomment and add the parameter details: srs.srsStorage.esCredentials.username and srs.srsStorage.esCredentials.password.
     # esCredentials:
     #   username: "username"
     #   password: "password"
     # To configure AWS IAM role-based authentication to your externally-managed Elasticsearch cluster, uncomment
     # and add the parameter details: srs.srsStorage.awsIAM and its associated region, srs.srsStorage.awsIAM.region
     # awsIAM:
-    #  region: "AWS_ELASTICSEARCH_REGION"
-    # To configure either authentication method, when the elasticsearch domain requires an open internet connection, uncomment to set this parameter to "true".
-    # requireInternetAccess: true
+    #   region: "AWS_ELASTICSEARCH_REGION"
+    # To configure either authentication method, when the elasticsearch domain requires an open internet connection,set requireInternetAccess parameter to "true".
```

Comment:
update for better punctuation>>
To configure either authentication method, when the Elasticsearch domain requires an open internet connection, set the requireInternetAccess parameter to "true".